test/e2e-cypress/static/documents/bugs/4641.yaml

```diff
@@ -0,0 +1,34 @@
+openapi: 3.0.0
+info:
+  title: Demo API
+  description: First test
+  termsOfService: 'http://demo.io/terms-of-service/'
+  contact:
+    name: Demo Support
+    email: support@demo.io
+  version: 1.0.0
+servers:
+  - url: 'https://httpstat.us/'
```

Comment:
Cypress tests need to avoid relying on external services - if `httpstat.us` is ever offline, this test will fail because Swagger UI will show `TypeError: failed to fetch` instead of the response content.

If you just drop the `servers` block, the request will just hit the test server and you'll get back a 404, which should work for your purposes since you only care about what's in the request 😄

Reply:
done in 3ef208e